src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_BOUND = 100;
```

Comment:
According to the checklist, avoid using static methods in your solution. Consider removing the `static` keyword from NUMBER_BOUND as it's not a method and doesn't need to be static in this context.